recipes/ckanweb-deploy.rb

```diff
@@ -37,10 +37,12 @@
 # Setup Site directories
 #
 storage_root = "#{shared_fs_dir}/ckan_storage"
+resource_cache = "#{shared_fs_dir}/resource_cache"
```

Comment:
I really don't like this being a blocking command. Could you please change this to chown the the root folder only.

The other issue is that this triggers on 'all' instances and we could have 5 of these running after each other and causing efs major load.